File: Data Structures & Algorithms/find-all-numbers-disappeared-in-an-array/submission-0.go
func findDisappearedNumbers(nums []int) []int {
	seenMap := make(map[int]struct{})
	result := make([]int,0)

	for _,val := range nums {
		seenMap[val] = struct{}{}
	}

	for i := 1; i <= len(nums); i++{
		if _,ok := seenMap[i]; !ok {
			result = append(result,i)
		}
	}

	return result
}

File: Data Structures & Algorithms/sort-array-by-increasing-frequency/submission-0.py
class Solution:
    def frequencySort(self, nums: List[int]) -> List[int]:
        counter = Counter(nums)
        result = []
        for key, value in sorted(counter.items(), key=lambda item:(item[1], -item[0])):
            for i in range(value):
                result.append(key)

        return result